libiocage/lib/JailConfig.py

```diff
@@ -79,8 +79,7 @@ def __init__(self,
 
         dict.__init__(self)
 
-        libiocage.lib.helpers.init_logger(self, logger)
-
+        self.logger = libiocage.lib.helpers.init_logger(logger)
```

Comment:
I like to use the output from the helper to store explicitly as self.logger, but we should remain passing self as first attribute. This way we do not need to pass other named arguments explicitly later on.